soca/obs/icefb_c2_esa.yaml

```diff
@@ -0,0 +1,42 @@
+- obs space:
+    name: icefb_c2_esa
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/icefb_c2_esa.{{window_begin}}.nc4
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/$(experiment).icefb_c2_esa.{{window_begin}}.nc4
+    simulated variables:  [sea_ice_freeboard]
+  obs operator:
+    name: SeaIceThickness
+  obs error:
+    covariance model: diagonal
+  obs filters:
+  - *obs_land_mask
+  # Remove obs in the Southerm Hemisphere
+  - filter: BlackList
+    where:
+    - variable:
+        name: latitude@MetaData
+      minvalue: -5
+      maxvalue: -90
```

Comment:
Is it better to change the range from minvalue: -60 to maxvalue: -90? I am not what could be the computational effect for larger range.

Reply:
@mjhossen updated